app/ui/legacy/src/main/res/layout/recipient_alternate_item.xml

```diff
@@ -143,6 +143,16 @@
             tools:visibility="visible"
             />
 
+        <ImageView
+            android:id="@+id/button_copy_email_address"
+            android:layout_width="48dp"
+            android:layout_height="48dp"
+            android:scaleType="center"
+            android:background="?attr/controlBackground"
+            android:contentDescription="@string/copy_action"
+            app:tooltipText="@string/copy_action"
```

Comment:
This doesn't seem to work on any of my devices. We do use `TooltipCompat.setTooltipText()` in a couple of places. So we know that works.

Reply:
Sure. I'll fix that